gluoncv/data/mscoco/detection.py

```diff
@@ -51,12 +53,13 @@ class COCODetection(VisionDataset):
 
     def __init__(self, root=os.path.join('~', '.mxnet', 'datasets', 'coco'),
                  splits=('instances_val2017',), transform=None, min_object_area=0,
-                 skip_empty=True):
+                 skip_empty=True, use_crowd=True):
```

Comment:
use_crowd by default should be False?

Reply:
Not sure. Current 0.2 release did not filter crowd instances.

Reply:
Okay, I think that should be fine, it depends on model as well.